server.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+
+var express = require('express');
+var bodyparser = require('body-parser');
+var Adjective = require('./lib/adjective.js')
+var Verb = require('./lib/verb.js')
+var Noun = require('./lib/noun.js')
+var BandName = require('./lib/bandname.js')
+var getRandomWord = require('./lib/getRandomWord.js')
+var postRandomWord = require('./lib/postRandomWord.js')
+var postBandName = require('./lib/postBandName.js')
+var app = express();
+var port = process.env.PORT || 3000;
+
+app.use(bodyparser.json());
+app.use(bodyparser.urlencoded({extended: true}));
+app.use(express.static(__dirname + '/app/'));
+
+var adjective = new Adjective();
+var verb = new Verb();
+var noun = new Noun();
+var bandname = new BandName();
+
+app.get('/', function(req, res) {
+  res.sendFile('index.html');
+});
+
+app.get('/adjective', function(req, res) {
+  res.json(getRandomWord(adjective));
+});
+
+app.get('/verb', function(req, res) {
+  res.json(getRandomWord(verb));
+});
+
+app.get('/noun', function(req, res) {
+  res.json(getRandomWord(noun));
+});
+
+app.post('/adjective', function(req, res) {
+  var word = postRandomWord(req.body.word, adjective);
+  res.json(word);
+});
+
+app.post('/verb', function(req, res) {
+  var word = postRandomWord(req.body.word, verb);
+  res.json(word);
+});
+
+app.post('/noun', function(req, res) {
+  var word = postRandomWord(req.body.word, noun);
+  res.json(word);
+});
+
+app.post('/bandname', function(req, res) {
```

Comment:
this is giving a 500 error when you hit it. If you open the "network" tab and look at the call to bandname, you get this:
TypeError: Converting circular structure to JSON<br> &nbsp; &nbsp;at Object.stringify (native)<br> &nbsp; &nbsp;at ServerResponse.json (/app/node_modules/express/lib/response.js:233:19)<br> &nbsp; &nbsp;at /app/server.js:56:7<br> &nbsp; &nbsp;at Layer.handle [as handle_request] (/app/node_modules/express/lib/router/layer.js:82:5)<br> &nbsp; &nbsp;at next (/app/node_modules/express/lib/router/route.js:110:13)<br> &nbsp; &nbsp;at Route.dispatch (/app/node_modules/express/lib/router/route.js:91:3)<br> &nbsp; &nbsp;at Layer.handle [as handle_request] (/app/node_modules/express/lib/router/layer.js:82:5)<br> &nbsp; &nbsp;at /app/node_modules/express/lib/router/index.js:267:22<br> &nbsp; &nbsp;at Function.proto.process_params (/app/node_modules/express/lib/router/index.js:321:12)<br> &nbsp; &nbsp;at next (/app/node_modules/express/lib/router/index.js:261:10)